src/components/rubric-levels-mobile.js

```diff
@@ -0,0 +1,1034 @@
+		} else if (dataIndex === 0 && this._selected !== -1) {
+			const radio = this._getRadioSibling(event.target, this._selected);
+			if (radio) {
+				this._checkRadio(radio);
+			}
+		} else {
+			this._updateFocused(dataIndex);
+		}
+	}
+
+	_handleKey(event) {
+		switch (event.code) {
+			case 'ArrowLeft':
+				return this._handleLeft(event);
+			case 'ArrowRight':
+				return this._handleRight(event);
+		}
+	}
+
+	_handleLeft(event) {
+		this._handleMove(event, -1);
+	}
+
+	_handleMouseDown(event) {
+		if (this._preventClick) {
+			return;
+		}
+
+		const dataIndex = parseInt(event.currentTarget.attributes.dataIndex.value);
+		this._handleDragStart(event, dataIndex);
+	}
+
+	_handleMouseOut() {
+		this._updateHovered(-1);
+	}
+
+	_handleMouseOver(event) {
+		const dataIndex = parseInt(event.currentTarget.attributes.dataIndex.value);
+		this._updateHovered(dataIndex);
+	}
+
+	_handleMove(event, delta) {
+		// Delay using afterNextRender so the _blurHandler in
+		// d2l-rubric-feedback will fire before the below statements.
+		afterNextRender(() => {
+			event.preventDefault();
+			event.stopPropagation();
+			this._moveSelection(delta);
+		});
+	}
+
+	_handleResize() {
+		this._width = this.clientWidth;
+	}
+
+	_handleRight(event) {
+		this._handleMove(event, 1);
+	}
+
+	_handleScroll(event) {
+		this._scroll = event.target.scrollLeft;
+	}
+
+	_handleThumbMouseDown(event) {
+		if (this._preventClick) {
+			return;
+		}
+		event.preventDefault();
+		this._handleDragStart(event, this._selected);
+	}
+
+	_handleThumbTransitionEnd() {
+		this._animating = false;
+	}
+
+	_isFocused(index, focused) {
+		return focused >= 0 && index === focused;
+	}
+
+	_isHovered(index, hovered) {
+		const isHovered =  hovered >= 0 && index === hovered;
+		if (isHovered) {
+			const hoveredLevel = this.assessmentMap[this._getSelfLink(this.criterionCells[index])];
+			if (hoveredLevel && hoveredLevel.properties && hoveredLevel.properties.score) {
+				this._updateHoveredScore(hoveredLevel.properties.score);
+			}
+		} else if (hovered < 0) {
+			this._updateHoveredScore(null);
+		}
+		return isHovered;
+	}
+
+	_isLeftIteratorActive(selected, focused, levels) {
+		const index = this._getIteratorIndex(selected, focused);
+		return this.readOnly
+			|| selected === -1 && focused === -1
+			|| (getComputedStyle(this).direction === 'rtl'
+				? index < levels.length - 1
+				: index > 0);
+	}
+
+	_isRightIteratorActive(selected, focused, levels) {
+		const index = this._getIteratorIndex(selected, focused);
+		return this.readOnly
+			|| selected === -1 && focused === -1
+			|| (getComputedStyle(this).direction === 'rtl'
+				? index > 0
+				: index < levels.length - 1);
+	}
+
+	_isSelected(index, selected) {
+		const isSelected = selected >= 0 && index === selected;
+		if (isSelected) {
+			const selectedLevel = this.assessmentMap[this._getSelfLink(this.criterionCells[index])];
+			if (selectedLevel && selectedLevel.properties && selectedLevel.properties.score) {
```

Comment:
```suggestion
			if (selectedLevel?.properties?.score) {
```
or use `RubricLevelEntity.score`